javascript/ql/lib/semmlecode.javascript.dbscheme

```diff
@@ -392,7 +392,7 @@ case @expr.kind of
 
 @exportspecifier = @named_export_specifier | @export_default_specifier | @export_namespace_specifier;
 
-@import_or_export_declaration = @import_declaration | @export_declaration;
+@import_or_export_declaration = @import_declaration | @export_declaration | @import_specifier;
```

Comment:
Could you rename `@import_or_export_declaration` to something more accurate, like `@type_keyword_operand`?